PWGDQ/Tasks/dileptonMuMu.cxx

```diff
@@ -286,13 +304,15 @@ struct DQEventMixing {
             continue;
           }
           VarManager::FillPair(muon1, muon2, fValues, VarManager::kJpsiToMuMu);
+          int centBin = fMixHandler->GetBinFromCategory(fMixHandler->GetMixingVariable("Cent"), event1.mixingHash());
```

Comment:
This line of code is just for your own debugging right? Otherwise i would avoid using this function in the mixing loop.

Reply:
It wasn't for debugging, I wanted to know in which centrality class I was to fill the adequate histogram directory, but it has been removed, due to your next comment